packages/aws-cdk-lib/aws-stepfunctions-tasks/test/athena/start-query-execution.test.ts

```diff
@@ -224,4 +224,138 @@ describe('Start Query Execution', () => {
       }),
     });
   });
+
+  test('execution parameters succeeds', () => {
```

Comment:
A couple of missing test cases:
1. `executionParameters: []`
2. `executionParameters: ['']`